certora/confs/ghostWithERC20s.conf

```diff
@@ -0,0 +1,21 @@
+// A default configuration for a contract using ERC20s
+{
+  // list of files that are part of the system 
+  "files": [
+    "src/contracts/Ghost.sol",
+    "certora/helpers/ERC20Helper.sol",
+    "certora/helpers/SimpleERC20.sol"
+  ],
+  // main contract to verify ghost, spec file 
+  "verify": "Ghost:certora/specs/moreExamplesERC20.spec",
+  // check the spec 
+  "rule_sanity" :"basic",
+  // assume 3 unroling of loops is enought 
```

Comment:
typo - enough.

I think some more elaboration here on loop iter or a link to docs
a suggestion - every loop will run for at most 3 iterations, this in turn will impose requirements on the length of the array (etc.) that a loop is iterating on

Reply:
done